### DISCUSSION TITLE
how to switch tabs

### DISCUSSION THREAD

**Discussion by manythrough** [LazyVim/LazyVim#2190] [Q&A] [answered]
hi there!

i have tried to switch tabs with gt and gT, but it is not working. am i doing something wrong?

also, everytime i press g, some window opens at the bottom. this is really annoying. can this be disabled?

**Comment by dpetka2001** [comment]
What you are seeing, are probably the buffers visually displayed as tabs by the plugin bufferline. If you do `<leader><Tab><Tab>` to create a new tab, then you will see at the top right two numbers (1 and 2). Those are the first and second Neovim tabs respectively. To move between buffers you can use `<S-h>` or `<S-l>`.
The window that pops up when you press a key is being provided by the plugin `which-key` and it shows you visual clues as to what combination of keybindings are available to you. You can disable it if you don't want this behavior.

**Reply by manythrough** [reply]
is there some way to get the most vim-like experience. i don't want to relearn everything. i like that  so much stuff is installed, but it would be easier to start from scratch.

**Reply by dpetka2001** [reply, marked as answer]
Then don't use a Neovim distro and start writing your own config. Most distros are opinionated. You can still disable plugins that you don't want. You could disable bufferline with the following file `~/.config/lua/plugins/disabled.lua`
```lua
return {
  { "akinsho/bufferline.nvim", enabled = false },
  -- add more plugins to disable here
}
```
It's up to you to decide if you start disabling plugins that you don't want or start writing your own config. Nobody knows what's best for you than yourself.

**Reply by dcoder32** [reply]
Thank you @dpetka2001 this <S-h> or <S-l> help me, I was having same issue.. know I can move between buffers in lazy an doom emacs. Thanks a lot!
